### ISSUE TITLE
upgrade latest esp8266 rtos sdk version

### ISSUE THREAD

**Issue by wuwx** [eerimoq/simba#59]
http://bbs.espressif.com/viewtopic.php?f=46&t=2349

**Comment by eerimoq** [comment]
Thanks for the information. Looks like there are no major updates but should update to the latest version of the RTOS anyway.

**Comment by simonlbn** [comment]
The latest SDK fixes the "KRAK" (https://www.krackattacks.com/) vulnerability in the wifi stack, so I think it would be a good idea to get the SDK updated.

The SDK is now at https://github.com/espressif/ESP8266_RTOS_SDK/commits/master so may make sense to switch to sub module import similar to esp32 SDK and more.

[I may have a go at this, but I want to finish the wifi scan work I'm currently working on first]

**Comment by eerimoq** [comment]
Yeah, its about time to make an uplift. It's not on my personal todo-list as of today, so any help would be appreciated (and probably needed to make it happen).
